clients/ranchercli/download.go

```diff
@@ -0,0 +1,99 @@
+package ranchercli
+
+import (
+	"fmt"
+	osys "os"
+	"os/exec"
+	"strings"
+
+	"github.com/rancher/shepherd/pkg/session"
+	"github.com/sirupsen/logrus"
+)
+
+type Client struct {
+	Session *session.Session
+}
+
+func (c *Client) Create(name string, args ...string) error {
+	err := c.ExecuteCommand(name, args...)
+	if err != nil {
```

Comment:
You don't need this if statement just return  c.ExecuteCommand(name, args...)

Reply:
Resolved